cloudformation/compute-fleet-hit-substack.cfn.yaml

```diff
@@ -605,6 +607,7 @@ Metadata:
   VPCId: !Ref 'VPCId'
   ClusterDNSDomain: !Ref 'ClusterDNSDomain'
   ResourcesS3Bucket: !Ref 'ResourcesS3Bucket'
+  ArtifactS3RootDirectory: !Ref 'ArtifactS3RootDirectory'
```

Comment:
minor: is this needed?

Reply:
I guess we can remove the entire `Metadata` section as well?